bpf/bpf_lxc.c

```diff
@@ -587,7 +587,12 @@ static inline int handle_ipv4_from_lxc(struct __sk_buff *skb, __u32 *dstID)
 			return ret;
 
 		cilium_dbg_capture(skb, DBG_CAPTURE_DELIVERY, HOST_IFINDEX);
+
+#ifdef POLICY_ENFORCEMENT_MODE
```

Comment:
@joestringer I'm open for suggestions to rename the C define, do you have anything in mind?

Reply:
A few ideas:
* `NETWORK_INTEGRATION_MODE`
* `CNI_INTEGRATION_MODE`
* `NON_ROUTING_MODE` (not so great, because even currently in direct routing mode we don't handle routing)
* `NETWORK_PLUMBING_MODE` (need to reverse the logic, though this is my favourite so far)

Reply:
change define to `HOST_REDIRECT_TO_INGRESS`